lean/commands/data/generate.py

```diff
@@ -106,13 +126,24 @@ def generate(start: datetime,
     lean_config_manager = container.lean_config_manager
     data_dir = lean_config_manager.get_data_directory()
 
+    if tickers:
+        tickers_options = ["--tickers=" + ",".join(tickers)]
+    else:
+        tickers_options = []
+
+    if tickers and symbol_count != len(tickers):
+        raise BadParameter(
+                f"--symbol-count ({symbol_count}) != number of tickers ({len(tickers)})"
+                )
+
     run_options = {
         "entrypoint": ["dotnet", "QuantConnect.ToolBox.dll",
                        "--destination-dir", "/Lean/Data",
                        "--app", "randomdatagenerator",
                        "--start", start.strftime("%Y%m%d"),
                        "--end", end.strftime("%Y%m%d"),
                        "--symbol-count", str(symbol_count),
+                       *tickers_options,
```

Comment:
> *tickers_options,

Suggest just using a string instead of an array since it's not required?

Reply:
didn't realize the list could have an empty string! that makes things easier/cleaner